week 1/lists_n_loops/index_search.py

```diff
@@ -0,0 +1,79 @@
+"""
+Group of functions that search all instances of a key in a list.
+Functions do same thing but with different looping mechanism.
+"""
+
+
+def search_key_first(nums: list[int], key: int) -> list[int]:
```

Comment:
instead of first, second, name the function with search method name. i.e. search_key_range.